### DISCUSSION TITLE
Unable to pair more than 20 devices - Solved

### DISCUSSION THREAD

**Discussion by agoralive** [Koenkk/zigbee2mqtt#6393] [General]
Do I need a zigbee2mqtt router to be able to pair more than 20 devices?

I have 20 devices connected, and it works just fine, but why can't I connect any more?

I have 1 coordinator, and 3 different devices which are discovered as routers - A Philips Lamp (not bulb), a Lidl LED Strip and an Ikea Socket device.

**Comment by m0wlheld** [comment]
> Do I need a zigbee2mqtt router to be able to pair more than 20 devices?

[Yes](https://www.zigbee2mqtt.io/information/FAQ.html#i-read-that-zigbee2mqtt-has-a-limit-of-20-devices-when-using-a-cc2531-is-this-true).

**Reply by agoralive** [reply]
Thank you
